VirtoCommerce.Storefront.IntegrationTests/Tests/ApiAccountControllerTests.cs

```diff
@@ -0,0 +1,205 @@
+    [CollectionDefinition("ApiAccountControlleTests", DisableParallelization = true)]
+    public class ApiAccountControllerTests : IClassFixture<StorefrontApplicationFactory>, IDisposable
+    {
+        private readonly HttpClient _client;
+        private bool _isDisposed;
+
+        public ApiAccountControllerTests(StorefrontApplicationFactory factory)
+        {
+            _client = factory.CreateClient();
+        }
+
+        [Fact(Skip = "TODO: need to check permissions")]
+        public async Task SearchOrganizationUsersAsync_GetOrganization()
+        {
+            //Arrange
+            _client.Login("admin", "store");
+            var request = CreateOrganizationRegistration();
+            await RegisterOrganizationAsync(request);
+            _client.Logout();
+            _client.Login(request.UserName, request.Password);
+
+            OrganizationContactsSearchCriteria searchCriteria = new OrganizationContactsSearchCriteria { PageSize = 1};
+            var content = new StringContent(
+                JsonConvert.SerializeObject(searchCriteria),
+                Encoding.UTF8,
+                "application/json");
+
+            //Act
+            var response = await _client.PostAsync($"{TestEnvironment.OrganizationEndpoint}/users/search", content);
+            var source = await response.Content?.ReadAsStringAsync();
+
+            var result = JsonConvert.DeserializeObject<UserSearchResult>(source);
+
+            //Assert
+            result.Should().BeEquivalentTo(new UserSearchResult());
+            _client.Logout();
+        }
+
+        [Fact]
+        public async Task RegisterOrganization_Registered()
+        {
+            //Arrange
+            _client.Login("admin", "store");
+
+            var request = CreateOrganizationRegistration();
+
+            //Act
+            var result = await RegisterOrganizationAsync(request);
+
+            //Assert
+            result.Succeeded.Should().BeTrue();
+            _client.Logout();
+        }
+
+        [Fact(Skip = "TODO: need to check permissions")]
+        public async Task UpdateOrganization_Updated()
+        {
+            //Arrange
+            
+            _client.Login("admin", "store");
+
+            var request = CreateOrganizationRegistration();
+            await RegisterOrganizationAsync(request);
+            _client.Logout();
+            _client.Login(request.UserName, request.Password);
+            
+            _client.Logout();
+            var organization = await GetCurrentOrganization();
+            _client.Login("admin", "store");
+            organization.Name += "after_update";
+            var content = new StringContent(
+                JsonConvert.SerializeObject(organization),
+                Encoding.UTF8,
+                "application/json");
+
+            //Act
+            await _client.PutAsync(TestEnvironment.OrganizationEndpoint, content);
+
+            //Assert
+            _client.Logout();
```

Comment:
can check the name was changed?
Also, should test address changing

Reply:
it's temporally skipped. there is a problem with permissions. need to resolve it